app/pages/sections/service-recipients/contextCreator.js

```diff
@@ -1,26 +1,56 @@
 import manifest from './manifest.json';
 import { baseUrl } from '../../../config';
 
-const getCheckedStatus = ({ selectStatus, data, selectedRecipientIdsData = [] }) => {
+const getCheckedStatus = ({ selectStatus, serviceRecipient, selectedRecipientIdsData = [] }) => {
   if (selectStatus === 'select') return true;
   if (selectStatus === 'deselect') return false;
   return !!selectedRecipientIdsData
-    .find(checkedRecipient => checkedRecipient.odsCode === data.odsCode);
+    .find(checkedRecipient => checkedRecipient.odsCode === serviceRecipient.odsCode);
 };
 
-const formatTableData = ({
-  selectStatus, serviceRecipientsData, selectedRecipientIdsData,
-}) => serviceRecipientsData
-  .map(data => ({
-    organisationName: {
-      id: data.odsCode,
-      name: data.odsCode,
-      value: data.name,
-      text: data.name,
-      checked: getCheckedStatus({ selectStatus, data, selectedRecipientIdsData }),
-    },
-    odsCode: data.odsCode,
-  }));
+const generateItems = ({
+  selectStatus,
+  serviceRecipientsData,
+  selectedRecipientIdsData,
+}) => {
+  const items = serviceRecipientsData.map((serviceRecipient) => {
+    const columns = [];
+    columns.push(({
+      question: {
+        dataTestId: `${serviceRecipient.name}-organisationName`,
+        checked: getCheckedStatus({ selectStatus, serviceRecipient, selectedRecipientIdsData }),
+        type: 'checkbox',
+        id: `${serviceRecipient.name}-organisationName`,
+        name: serviceRecipient.odsCode,
+        value: serviceRecipient.name,
+        text: serviceRecipient.name,
+      },
+      dataTestId: `${serviceRecipient.name}-organisationName`,
+      classes: 'nhsuk-u-font-size-12',
```

Comment:
This class should be coming from the manifest. A cellInfo details should provide this.